src/main/java/org/datacleaner/kettle/configuration/DataCleanerConfigurationDialog.java

```diff
@@ -184,7 +183,6 @@ public void widgetSelected(SelectionEvent event) {
             }
         } catch (Throwable e) {
             // Do nothing If the file doesn't exit 
-
```

Comment:
The story was this one https://github.com/datacleaner/pdi-datacleaner/issues/10. 
That exception happens if DC was never configured before, and there is no configuration to show. Even if by mistake the file is not there by any other cause, it would be weird to show a message with "File not found" exception in a configuration dialog. And anyways the user can set it again, and the file will be recreated. 

@LosD  I wasn't aware I cannot make any short of decision, especially for a lousy configuration dialog.